transports/http/endpoints/api/merkleroots/endpoints.go

```diff
@@ -25,6 +32,39 @@ func (h *handler) RegisterAPIEndpoints(router *gin.RouterGroup, _ *config.HTTPCo
 	merkle := router.Group("/chain/merkleroot")
 	{
 		merkle.POST("/verify", h.verify)
+		merkle.GET("", h.merkleroots)
+	}
+}
+
+// Merkleroot godoc.
+//
+// @Summary Returns merkleroots for the specified range.
+// @Tags merkleroots
+// @Accept */*
+// @Produce json
+// @Success 200 {object} domains.MerkleRootsESKPagedResponse
+// @Router /chain/merkleroot [get]
+// @Param batchSize query string false "Batch size of returned merkleroots"
+// @Param lastEvaluatedKey query string false "Last evaluated merkleroot that client has processed"
+// @Security Bearer
+func (h *handler) merkleroots(c *gin.Context) {
+	batchSize := c.DefaultQuery("batchSize", defaultBatchSize)
+	lastEvaluatedKey := c.Query("lastEvaluatedKey")
+
+	batchSizeInt, err := strconv.Atoi(batchSize)
+	if err != nil || batchSizeInt < 0 {
+		err, statusCode := helpers.ErrorResponse(domains.ErrMerklerootInvalidBatchSize)
```

Comment:
Maybe for the future - byt I don't really like the name `helpers` for it.

Reply:
yes, it could be for example `response`
`response.Error(....)`

Reply:
changed to response for now